.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "nixpkgs",
+  "image": "mcr.microsoft.com/devcontainers/universal:2-linux",
+  "features": {
+    "ghcr.io/devcontainers/features/nix:1": {
+      "multiUser": false
+    }
+  },
```

Comment:
It would be nice to add VS Code settings to set up Nix-specific IDE extensions.

```json
	"customizations": {
		"vscode": {
			"extensions": [
				"bbenoist.Nix",
				"jnoortheen.nix-ide"
			],
			"settings": {
				"nix.formatterPath": "nixpkgs-fmt",
				"nix.enableLanguageServer": true,
				"nix.serverPath": "nixd"
			}
		}
	},
```